web/src/main/react-dashboard/src/components/DeploymentPatternView.js

```diff
@@ -43,18 +43,27 @@ class DeploymentPatternView extends Component {
     };
   }
 
+  handleError(response) {
+    if (!response.ok) {
+      throw Error(response.statusText)
```

Comment:
Should we log the error to the console. What's the best practice in JS world?

Reply:
We can use react error boundaries.

[1] https://blog.bugsnag.com/react-16-error-handling/
[2] https://reactjs.org/blog/2017/07/26/error-handling-in-react-16.html

Reply:
Used 'catch' phrase and logging as per [1]

[1] - https://www.tjvantoll.com/2015/09/13/fetch-and-errors/